src/components/StaticThought.tsx

```diff
@@ -136,6 +137,7 @@ const StaticThought = ({
     _.isEqual,
   )
 
+  const dragInProgress = useSelector(state => isTouch && isSafari() && state.dragInProgress)
```

Comment:
There are other places in the codebase where we define `dragInProgress` as simply `useSelector(state => state.dragInProgress)`, so we should probably call this one `dragInProgressSafari` to avoid confusion.